pkg/sampleoperator/sampleproduces/sample_apply_configuration.go

```diff
@@ -0,0 +1,39 @@
+package sampleproduces
+
+import (
+	"context"
+	"github.com/openshift/multi-operator-manager/pkg/library/libraryproduces"
+)
+
+func SampleRunApplyProduces(ctx context.Context) (*libraryproduces.ProducedResources, error) {
+	// TODO probably make this a yaml file directly?  I don't know, it's not too bad like this.
+	return &libraryproduces.ProducedResources{
+		ConfigurationServerResources: libraryproduces.ResourceList{},
+		ManagementServerResources: libraryproduces.ResourceList{
```

Comment:
I'm slightly confused because, in the description of this PR, you say that we need to define the resources we are going to write, but the list looks like the resources we are going to read/watch.

Reply:
> I'm slightly confused because, in the description of this PR, you say that we need to define the resources we are going to write, but the list looks like the resources we are going to read/watch.

It's very similar.  We could even discuss if we should automatically include the produced resources in the input to the apply-configuraiton command.  But write permissions are required on produces.